rocksdb/rocksdb.nim

```diff
@@ -465,6 +465,22 @@ proc releaseSnapshot*(db: RocksDbRef, snapshot: SnapshotRef) =
     rocksdb_release_snapshot(db.cPtr, snapshot.cPtr)
     snapshot.setClosed()
 
+proc flush*(db: RocksDbRef, cfs: openArray[ColFamilyHandleRef]): RocksDBResult[void] =
+  withLock(db.lock):
+    if not db.isClosed():
```

Comment:
I would make this isClosed check an assertion because we should never try flushing a closed database.